modules/nf-core/openms/peptideindexer/tests/main.nf.test

```diff
@@ -0,0 +1,78 @@
+nextflow_process {
+
+    name "Test Process OPENMS_PEPTIDEINDEXER"
+    script "../main.nf"
+    process "OPENMS_PEPTIDEINDEXER"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "openms"
+    tag "openms/decoydatabase"
+    tag "openms/peptideindexer"
+
+    setup {
+        run("OPENMS_DECOYDATABASE") {
+            script "../../decoydatabase/main.nf"
+            process {
+                """
+                input[0] = [
+                    [id:'test'],
+                    file(params.test_data['proteomics']['database']['yeast_ups'], checkIfExists: true)
```

Comment:
New way of writing file paths now, instead of this:
file(params.test_data['proteomics']['openms']['idxml1'], checkIfExists: true)
do this:
file(params.modules_testdata_base_path + 'proteomics/openms/HepG2_rep1_small.idXML', checkIfExists: true)

Reply:
Can you change this in all inputs?